Library/Homebrew/test/.codecov.yml

```diff
@@ -1,7 +0,0 @@
-comment: false
-coverage:
-  fixes:
-    - "::Library/Homebrew/"
-  parsers:
-    v1:
-      include_full_missed_files: yes
```

Comment:
That appears to only apply to [v1.0 of Codecov](https://docs.codecov.io/v1.0/docs/language-configuration), and the current version is 4.3.6.